smallWidgets/TvShowTreeView.cpp

```diff
@@ -225,6 +225,9 @@ void TvShowTreeView::drawRowBackground(QPainter *painter, QStyleOptionViewItem o
         if (alternatingRowColors()) {
             if (index.row() % 2 == 0) {
                 option.features |= QStyleOptionViewItem::Alternate;
+            #ifdef Q_OS_WIN
+                option.features &= ~QStyleOptionViewItem::Alternate;
```

Comment:
Seems like you set the style option and unset it again if you're on windows? Wouldn't it make more sense to only call `option.features |= QStyleOptionViewItem::Alternate;` if `#ifndef Q_OS_WIN`? :)